repository/sql/repository.go

```diff
@@ -132,13 +132,13 @@ func (r *repositoryGorm) FirstOrCreate(entity repository.Model, filters ...repos
 //
 //	filters: selection criteria for entries that should be considered when counting entries.
 func (r *repositoryGorm) Count(filters ...repository.Filter) (uint64, error) {
-	var count uint64
+	var count int64
```

Comment:
When would this value be less than 0? Should we return an error if that is the case? 

Gorm might return an error whenever it's less than 0 but I'd like to double check.

Reply:
> When would this value be less than 0? Should we return an error if that is the case?

GORM's `Count` method expects a pointer to an `int64`, that's the reason for this change.